### ISSUE TITLE
Firebase authentication error

### ISSUE THREAD

**Issue by reveurguy** [firebase/codelab-friendlychat-web#605]
**Describe the bug**
I am following the Codelab instructions for a friendly chat app . When i do firebase login , the link shown in terminal opens a page where it shows: 
Authorization Error
Error 400: invalid_request
Required parameter is missing: response_type

This is the message i get and thus i can't do the further steps.

**To Reproduce**
https://firebase.google.com/codelabs/firebase-web#3
I am getting the error in step 3 here.

**Expected behavior**

**Additional context**

**Comment by JamesVanWaza** [comment]
@reveurguy were you able to successfully set up your project on the https://console.firebase.google.com , if you did your credentials should work fine when you run `firebase login`

**Comment by reveurguy** [comment]
> @reveurguy were you able to successfully set up your project on the https://console.firebase.google.com, if you did your credentials should work fine when you run `firebase login`

yes, the project was set up successfully, but when i open the link on running `firebase login`, i get the above mentioned error.

**Comment by JamesVanWaza** [comment]
@samtstern any ideas?

**Comment by samtstern** [comment]
Some people have mentioned this issue on the Firebase CLI repo:
https://github.com/firebase/firebase-tools/issues?q=is%3Aissue+400+response_type+

It seems that this comes from the login URL being truncated in your browser so that it's missing the `code` param.

**Comment by Wasabi95** [comment]
I got the same error, got stuck in the authentication .

**Comment by JamesVanWaza** [comment]
@Wasabi95, which version of firebase are you running?

**Comment by Wasabi95** [comment]
10.0.1

> @Wasabi95, which version of firebase are you running?

10.0.1

**Comment by JamesVanWaza** [comment]
@Wasabi95,  try copying the full url, see if that works. mine works on `10.0.1`, also post a screenshot for the error. @samtstern might be able to assist.

**Comment by Wasabi95** [comment]
thanks, I will post it

**Comment by Wasabi95** [comment]
Do you know how to preview a react project on GitHub pages?  I follow the instructions here https://github.com/gitname/react-gh-pages but it did not work

**Comment by Wasabi95** [comment]
I followed all the steps but I got stuck in step number 8 Scale with Cloud SQL when creating file " locations.go " error error loading workspace you are outside of a module. what might be the mistake
![2022-01-08](https://user-images.githubusercontent.com/74787751/148648809-13f3de31-785e-4657-874b-7fbceb33eda9.png)
?

**Comment by JamesVanWaza** [comment]
Which tutorial link are you trying to set up?

**Comment by Wasabi95** [comment]
I gor an erros in this codelab 
https://developers.google.com/codelabs/maps-platform/full-stack-store-locator?hl=en#0

and also this one 
https://firebase.google.com/codelabs/firebase-web?hl=en#0
here I got the sign in error

**Comment by Wasabi95** [comment]
> 

can you help me out with these 2 projects please ??

**Comment by JamesVanWaza** [comment]
Will try and help with the Firebase one, the other one I haven't checked it out. Once I get to the laptop will retest and see how to solve it.

**Comment by Wasabi95** [comment]
thanks

**Comment by JamesVanWaza** [comment]
@Wasabi95 let's create some time to work on it together, let me know your availability, send me an email at info@dosbranding.com so we can close this ticket.

**Comment by Wasabi95** [comment]
Hello I replied to email provided by you, it wasnt delivered. anyway let me know when you are available, thanks

**Comment by BlackCagouProd** [comment]
I have a problem with auth in the user interface, it does not return anything as soon as I click on signs-in, the popup window opens but does not display any form to connect, no error message is returned too
